### PR TITLE
Move some GPTQ functions to common

### PR DESCRIPTION
**Pull request by eladc-git**

### CODE REVIEW

File: model_compression_toolkit/gptq/common/gptq_training.py [hunk 1]
```diff
@@ -63,6 +65,104 @@ def __init__(self,
 
         self.fxp_model, self.gptq_user_info = self.build_gptq_model()
 
+    def get_optimizer_with_param(self,
+                                 flattened_trainable_weights,
+                                 flattened_bias_weights,
+                                 trainable_quantization_parameters,
+                                 temperature_weights):
```
Comment (reuvenperetz) [inline model_compression_toolkit/gptq/common/gptq_training.py:72]: Please add comments and type hints

**Review by reuvenperetz — CHANGES_REQUESTED** [review]

File: model_compression_toolkit/gptq/common/gptq_training.py (hunk 1 above)
Comment (eladc-git) [reply model_compression_toolkit/gptq/common/gptq_training.py]: done

**Review by reuvenperetz — APPROVED** [review]

**Comment by haihabi** [comment]
I am not sure about this PR since there is a significant difference between Keras and PyTorch optimizers.
Let's talk about it. Maybe we need to change Keras API.

**Comment by eladc-git** [comment]
> I am not sure about this PR since there is a significant difference between Keras and PyTorch optimizers. Let's talk about it. Maybe we need to change Keras API.

This code is only optimizer object and weights holder so it is not supposed to be framework related.

**Pull request merged**